packages/block-library/src/group/editor.scss

```
@@ -39,9 +39,9 @@
 	&::after {
 		content: "";
 		display: flex;
-		flex: 1 0 $grid-unit-60;
+		flex: 1 0 $grid-unit-50;
```

The values are the same, but let's use this variable instead so it's more meaningfully aligned with the button height.

```suggestion
		flex: 1 0 $button-size-next-default-40px;
```